### PR TITLE
Google Data Studio example

### PR DESCRIPTION
**Pull request by KlippRid**
This tutorial aims to show a user how to create a dashboard. It leaves out steps such as setting up an endpoint and creating a database. The focus is instead how to in an easy fashion visualize the data at hand.

In future we could also create Wittra templates for customers to use.

### CODE REVIEW

File: data-studio/data-studio.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+![](images/example_dashboard.png)
+
+This is a short tutorial on how to set up a dashboard (or report as stated by
+data studio) in Google Data Studio (this can easily be done in other tools such
+as [Tableu](https://www.tableau.com/)).
+
+The aim is to show the potential of what you can do with the data from our
+Wittra sensor network.
+
+### Prerequisits
```
Comment (drageryd) [inline data-studio/data-studio.md]:
```suggestion
### Prerequisites
```

File: data-studio/data-studio.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+In order to produce the dashboard it assumes you have already set up a
+database, to store the streamed data from the network. In this example we use
+[BigQuery](https://cloud.google.com/bigquery/what-is-bigquery), an enerprise
+data warehouse by Google.
+
+A Google account, to log into Data Studio. Note that Data Studio is free, but
+the queries to the data base may cost money depending on the database ad the
```
Comment (drageryd) [inline data-studio/data-studio.md]:
```suggestion
the queries to the data base may cost money depending on the database and the
```

File: data-studio/data-studio.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+temperature and usage. We will choose **BigQuery** as data source and retrieve
+the data with a **custom query**.
+
+```
+SELECT
+  *
+FROM
+  `<project-id.dataset.table>`
+where tag_id like '<tag-id>'
+order by timestamp desc
```
Comment (drageryd) [inline data-studio/data-studio.md]: Not an SQL expert but I would have the SQL keywords in caps, something like this:
```suggestion
SELECT *
FROM <project-id.dataset.table>
WHERE tag_id like <tag-id>
ORDER BY timestamp DESC
```
How would you add a "link" to a database outside the Google echosystem? The `<project-id.dataset.table>` assumes that your account can access the GCP project directly?
Then maybe add something like: "Replace `<tag-id>` with one of the device IDs in your project."

File: data-studio/data-studio.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+## Create reports and dashboards with Data Studio
```
Comment (drageryd) [inline data-studio/data-studio.md]: Can this be renamed to `README.md`? Then GitHub will display it when you enter the directory and you wont have to click the markdown file?

File: data-studio/data-studio.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+The aim is to show the potential of what you can do with the data from our
+Wittra sensor network.
+
+### Prerequisits
+
+In order to produce the dashboard it assumes you have already set up a
+database, to store the streamed data from the network. In this example we use
+[BigQuery](https://cloud.google.com/bigquery/what-is-bigquery), an enerprise
+data warehouse by Google.
```
Comment (wegelid-wittra) [inline data-studio/data-studio.md]: Setting up the database is a pretty big step. We could point the users in the direction of Google provided solutions, such as [this one](https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#running-the-pubsub-topic-to-bigquery-template).

File: data-studio/README.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+as [Tableu](https://www.tableau.com/)).
+
+The aim is to show the potential of what you can do with the data from our
+Wittra sensor network.
+
+### Prerequisites
+
+In order to produce the dashboard it assumes you have already set up a
+database, to store the streamed data from the network. In this example we use
+[BigQuery](https://cloud.google.com/bigquery/what-is-bigquery), an enerprise
```
Comment (LDVG) [inline data-studio/README.md]:
```suggestion
[BigQuery](https://cloud.google.com/bigquery/what-is-bigquery), an enterprise
```

File: data-studio/README.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+data warehouse by Google. For inspiration of of how to setup a database from
+streamed data please see [this example](https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#running-the-pubsub-topic-to-bigquery-template)
```
Comment (LDVG) [inline data-studio/README.md]:
```suggestion
streamed data please see [this example](https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#running-the-pubsub-topic-to-bigquery-template).
```

File: data-studio/README.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+### Prerequisites
+
+In order to produce the dashboard it assumes you have already set up a
+database, to store the streamed data from the network. In this example we use
+[BigQuery](https://cloud.google.com/bigquery/what-is-bigquery), an enerprise
+data warehouse by Google. For inspiration on how to setup a database from
+streamed data please see [this example](https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#running-the-pubsub-topic-to-bigquery-template)
+
+A Google account, to log into Data Studio. Note that Data Studio is free, but
+the queries to the data base may cost money depending on the database and the
```
Comment (LDVG) [inline data-studio/README.md]:
```suggestion
the queries to the database may cost money depending on the database and the
```

File: data-studio/README.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+automatically, if not click `add data` to the right in the top panel.
+
+![](images/datastudio_add_data.png)
+
+There are several different type of connectors already existing in Data Studio,
+some are directly connected to the Google infrastructure and some are external
+connectors such as *MySQL*, *PostgreSQL* and *File Upload*. You
+can also [add your own connectors](https://developers.google.com/datastudio/connector/build)
+to interface with Data Studio.
+In this example we will create a dashboard for a specific tag showing
```
Comment (LDVG) [inline data-studio/README.md:47]: Either break the line once more to create a new paragraph, or concatenate with previous line to keep it in one. :slightly_smiling_face:

File: data-studio/README.md [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+
+![](images/breakdown_time.gif)
+
+### 5. Refreshing data
+
+The data shown on the dashboard is pulled from a database, the default
+frequency on how often the data is pulled is 12 hours. You can change this to
+either every hour or every 4th hour, in BigQuery how often you pull is creating
+cost, by opening **Resource>Manage** added data sources>edit. In the top panel
+click the `Data freshness` button and change it to the prefered data frequency.
```
Comment (LDVG) [inline data-studio/README.md]:
```suggestion
click the `Data freshness` button and change it to the preferred data frequency.
```

**Review by LDVG — APPROVED** [review]
Looks very cool!

**Review by jerrymek — APPROVED** [review]

**Pull request merged**